Compilation errors on Windows 10 with Visual Studio 2019

**Godot version:**
Master 8ffa35e44f93321d618b81ec48817f5de8668a83

**OS/device including version:**
Windows 10
Visual Studio Community 2019, version 16.4.5

**Issue description:**
This error appears during compilation:
C:\Program Files (x86)\Windows Kits\10\include\10.0.10586.0\um\combaseapi.h(234): error C2760: syntax error: unexpected token 'identifier', expected 'type specifier'

**Steps to reproduce:**
n/a

**Minimal reproduction project:**
n/a

--- a/SConstruct
+++ b/SConstruct
@@ -330,7 +330,7 @@ if selected_platform in platform_list:
     else:
         # MSVC doesn't have clear C standard support, /std only covers C++.
         # We apply it to CCFLAGS (both C and C++ code) in case it impacts C features.
-        env.Prepend(CCFLAGS=['/std:c++17', '/permissive-'])
+        env.Prepend(CCFLAGS=['/std:c++17'])
 
     # Enforce our minimal compiler version requirements
     cc_version = methods.get_compiler_version(env) or [-1, -1]

